### PR TITLE
CMR-5219 Updates api docs

### PR DESCRIPTION
**Pull request by daniel-zamora**

### CODE REVIEW

File: search-app/docs/api.md [hunk 1]
```diff
@@ -1599,6 +1599,10 @@ Find an additional attribute with name "X\Y\Z" with value 7.
 
     curl "%CMR-ENDPOINT%/collections?attribute\[\]=float,X\Y\Z,7"
 
+Find an additional attribute with name "MISSION_NAME" with value "Big Island, HI".
+
+    curl "%CMR-ENDPOINT%/collections?attribute\[\]=string,MISSION_NAME,Big Island\, HI"
```
Comment (CayvonH) [inline search-app/docs/api.md]: Should the space in Big Island be URL encoded?

Comment (ygliuvt) [reply search-app/docs/api.md]: We should probably add a test for this in system-int-test if there is not one already.

Comment (daniel-zamora) [reply search-app/docs/api.md]: They aren't encoding the commas in the docs, so I didn't encode the space.

Comment (daniel-zamora) [reply search-app/docs/api.md]: There are already tests for escaping commas for additional attributes.

Comment (chris-durbin) [reply search-app/docs/api.md]: Oh show me those tests - that was the first thing I looked for when Greg reported the problem.

**Pull request merged**